tests/regression/test_interpolate.py

```diff
@@ -434,7 +434,7 @@ def test_interpolate_minmax(access):
     g = interpolate(x*y, V)
     f = interpolate(x**2 - y**4, V)
     actual = Function(g)
-    actual = interpolate(f, actual, access=access).dat.data_ro
+    actual = Interpolator(f, actual, access=access)._interpolate().dat.data_ro
```

Comment:
This is another place where we need the old API to test min/max. If this is abuse of notation as @dham suggested, perhaps different accessors should just be removed.

Reply:
Either that or let `access` be a kwarg to `Interpolate`